.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": false
```

Comment:
Left this in to prevent future VSCode users from accidentally autoformatting (I certainly tripped on it.) You're free to remove it from the pr if you want.

Reply:
It seems fine to leave it, makes sense.